README.md

```diff
@@ -78,9 +78,9 @@ Modern software development practices _assume_ support for reviewing changes, tr
 
 ## Tutorials
 
-- [Managing Helm releases the GitOps way](https://github.com/fluxcd/helm-operator-get-started) - Flux and Helm Operator tutorial
+- [Managing Helm releases the GitOps way](https://fluxcd.io/docs/components/source/helmrepositories/#spec-examples) - Flux and Helm Operator tutorial
```

Comment:
Please link to https://github.com/fluxcd/flux2-kustomize-helm-example

Reply:
done